virtual-host-gatherer/lib/gatherer/modules/File.py

```diff
@@ -86,12 +90,12 @@ def run(self):
         if not urlparse.urlsplit(self.url).scheme:
             self.url = "file://%s" % self.url
         try:
-            output = json.loads(urlgrabber.urlread(str(self.url), timeout=300))
+            output = json.load(urlopen(str(self.url), timeout=300))
```

Comment:
As I had so many bad experience with http implementations I have to ask how urllib behave with SSL, IPv6, Proxy and authentication?

Reply:
E.g. You are in an IPv6 only environment, the target support only SSL, you must use a HTTP Proxy and that requires NTLM authentication. Will it work?

Reply:
Hey @mcalmer - Thanks for pointing this out here!

I've finally replaced `python-urlgrabber` with `python-pycurl` [1] which is actually the recommended replacement on Fedora [2]. `pycurl` supports most of environments including of course IPv6, HTTPS, Proxy and NTLM.

I think this is a better choice. What do you think?

[1] - http://pycurl.io/docs/latest/
[2] - http://fedora.portingdb.xyz/pkg/python-urlgrabber/

Reply:
Yes, "curl" works everywhere :-)
Thanks.